.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+---
+env:
+  - CGO_ENABLED=0
+  - GO111MODULE=on
+before:
+  hooks:
+    - go mod download
+builds:
+  - id: project
+    main: ./cmd/trivy-db
```

Comment:
Adding release, the test is [here](https://github.com/krol3/trivy-db/releases/tag/v1.1.5)

Reply:
It is unnecessary to release the trivy-db binary. We should release only the database file.